20190704/AGENDA.md

```diff
@@ -0,0 +1,15 @@
+# Sardana Follow-up Meeting - 2019/07/04
+
+To be held on 2019/07/04 at 14:00
+
+## Agenda
+
+1. Presentation - any idea?
```

Comment:
IMO the more time we dedicate to SEP16 the better. I would for example discuss in detail the cookiecutter template that @amilan proposed in the last meeting.